src/CONST.js

```diff
@@ -847,7 +847,11 @@ const CONST = {
         ITEM_HEIGHT: 40,
         SMALL_CONTAINER_HEIGHT_FACTOR: 2.5,
         MIN_AMOUNT_OF_ITEMS: 3,
-        MAX_AMOUNT_OF_ITEMS: 5,
+        // max number of suggestions
+        MAX_AMOUNT_OF_ITEMS: 20,
+        // used for limiting the height of suggestions container, we show a scrollable list
+        // with max height equal to MAX_AMOUNT_OF_VISIBLE_ITEMS_IN_WINDOW (5) * ITEM_HEIGHT.
+        MAX_AMOUNT_OF_VISIBLE_ITEMS_IN_WINDOW: 5,
```

Comment:
Similarly, with a better constant name, I think we can obviate the need for the comment. How about: 
```suggestion
        MAX_AMOUNT_OF_VISIBLE_SUGGESTIONS_IN_CONTAINER: 5,
```